aion/web/elm/src/Update.elm

```diff
@@ -121,32 +140,32 @@ update msg model =
             else
                 model ! []
 
-        NoOperation ->
-            model ! []
-
-        SetNewQuestionContent questionContent ->
+        UpdateCreateQuestionForm name value ->
             let
                 oldPanelData =
                     model.panelData
             in
-                { model | panelData = { oldPanelData | newQuestionContent = questionContent } } ! []
+                { model
+                    | panelData =
+                        { oldPanelData | createQuestionForm = Forms.updateFormInput oldPanelData.createQuestionForm name value }
+                }
+                    ! []
 
-        SetNewAnswerContent answerContent ->
+        CreateNewQuestionWithAnswers ->
             let
-                oldPanelData =
-                    model.panelData
+                possibleFields =
+                    [ "question", "answers", "subject" ]
+
+                validationErrors =
+                    possibleFields
+                        |> List.map (\name -> Forms.errorList model.panelData.createQuestionForm name)
```

Comment:
For me it works as the Elixir's style of aliasing nested modules - I would put `let createQuestionForm = model.panelData.createQuestionForm` to get it of this dot army.